components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -0,0 +1,122 @@
+
+Mention the correct Name, FQDN, and IP Address of the system. Click on **Add Chef Server**. The desired server gets added to the Chef Infra Server.
+
+**Note:** If the mentioned FQDN and IP Address is incorrect, the box displays an error as shown below:
+
+![Add Chef Server Popup Menu](/images/automate/add-chef-server-popup-menu-with-error.png)
+
+Click on the listed Servers to view the details of the specific **Chef Server**.
+
+## Connect to the Chef Infra Server Organization
+
+Selecting a specific added Chef Server opens the list of organizations in the Server. Refer to the image below:
+
+![Chef Infra Server Organization](/images/automate/chef-server-organization.png)
+
+The Organization in the Chef server listed are the ones which are already created in that specific server. To create a new Organization in the Server click [here](https://docs.chef.io/automate/infra_server/#set-up-the-chef-infra-server).
+
+To add the previously created organization to the Infrastructure, click on the **Add Chef Organization** button. Refer to the image below:
+
+![Add Chef Organization](/images/automate/add-chef-organization-button.png)
+
+Clicking on the **Add Chef Organization** button opens a popup menu as shown below:
+
+![Add Chef Organization Popup Menu](/images/automate/add-chef-organization-popup-menu.png)
+
+Add the Name, Projects, Admin User, and Admin Key. Always use the exact details that were provided at the time of setting up **Knife**. Copy the contents of the `~/.chef/USER.pem` file and past it into the **Admin Key** field. Once done click **Add Chef Organization**. The desired organization will be added to the Chef Server.
+
+Click on any Chef Organization to view the **Objects**.
+
+## Access Chef Infra Server Objects
+
+The **Chef Infra Server Objects** can be managed from the Chef Automate. In Automate Infra views, following objects can be managed:
+
+- Cookbooks
+- Roles
+- Environments
+- Data Bags
+- Clients
+
+Refer to the image below:
+
+![Chef Infra Server Objects](/images/automate/chef-infra-server-objects.png)
+
+### Cookbooks
+
+A cookbook is the fundamental unit of configuration and policy distribution. Cookbooks also comprise recipes and other optional components or directories. Click [here](https://docs.chef.io/cookbooks/) to know more about Cookbooks.
+
+### Roles
+
+A role is a way to define certain patterns and processes that exist across nodes in an organization as belonging to a single job function. Each role consists of zero (or more) attributes and a run-list. Click [here](https://docs.chef.io/roles/) to know more about Roles.
+
+### Environments
+
+An environment is a way to map an organization’s real-life workflow to what can be configured and managed when using Chef Infra. This mapping is accomplished by setting attributes and pinning cookbooks at the environment level. Click [here](https://docs.chef.io/environments/) to know more about Environments.
```

Comment:
I think we should avoid "is a way" if we can. 
Use named links instead of `[here]`